ansible_collections/arista/avd/roles/eos_designs/doc/l3ls-evpn/network-services.md

```diff
@@ -45,7 +45,9 @@ svi_profiles:
   < profile_name >:
     mtu: < mtu >
     enabled: < true | false >
-    ip_virtual_router_address: < IPv4_address/Mask >
+    ip_virtual_router_addresses:
+      - < IPv4_address/Mask >
```

Comment:
Note there is a big difference in EOS between `ip virtual-router address <ip / mask>` and `ip virtual-router address <ip>`. The template currently has a filter that removes any mask, so either we should only show "IPv4_address" here or we change the template to _not_ extract the IP alone, and then show both options `< IPv4_address | IPv4Address/Mask >` here.

Reply:
My bad, it was a wrong description. I will remove /Mask as it should not have any mask.